homeassistant/components/atag/strings.json

```diff
@@ -12,11 +12,11 @@
       }
     },
     "error": {
-      "unauthorized": "Pairing denied, check device for auth request",
```

Comment:
I'm not sure i would change this one, the message meaning is not the same.

Reply:
I agree, let's revert this change.